entries/e/elster.de.json

```
@@ -1,11 +1,19 @@
 {
   "Elster": {
     "domain": "elster.de",
-    "url": "https://www.elster.de/",
     "tfa": [
       "custom-software",
       "custom-hardware"
     ],
+    "custom-hardware": [
+      "National identity card",
```

Would this be a fair translation?
```suggestion
      "eID card",
```

---

The same name was already used for "DATEV", "Deutsche Rentenversicherung" and "Hamburg Serviceportal". The "eID" is actually a function of the "ePA" (_elektronischer Personalausweis_, i.e. electronic identity card).